testing/sdk_networks.py

```diff
@@ -1,10 +1,18 @@
+import json
+
 import shakedown
 
 
 def check_task_network(task_name, expected_network_name="dcos"):
     """Tests whether a task (and it's parent pod) is on a given network
     """
     _task = shakedown.get_task(task_id=task_name, completed=False)
+
+    if type(_task) == list:
```

Comment:
According to the docstring it is *never* a list, but a tuple. If you did want to do a type test, you should use isinstance()